environment-creator-application/src/main/java/com/ca/apim/gateway/cagatewayconfig/EnvironmentCreatorApplication.java

```diff
@@ -27,11 +28,17 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import static com.ca.apim.gateway.cagatewayconfig.KeystoreCreator.createKeyStoreIfNecessary;
+
 public class EnvironmentCreatorApplication {
+
     private static final Logger logger = Logger.getLogger(EnvironmentCreatorApplication.class.getName());
+    private static final String SYSTEM_PROPERTIES_PATH = "/opt/SecureSpan/Gateway/node/default/etc/conf/system.properties";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor your code to get this URI from a customizable parameter. ([squid:S1075](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1075))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=76&issues=AWZ5IarqP8lmccBDJdSo&open=AWZ5IarqP8lmccBDJdSo)

[ISSUE_KEY]: AWZ5IarqP8lmccBDJdSo